playbooks/common/openshift-cluster/upgrades/upgrade.yml

```diff
@@ -203,7 +215,7 @@
     command: >
       {{ openshift.common.admin_binary }} manage-node {{ openshift.common.hostname | lower }} --schedulable=true
     delegate_to: "{{ groups.oo_first_master.0 }}"
-    when: inventory_hostname in groups.oo_nodes_to_config and openshift.node.schedulable | bool
```

Comment:
This was coming from openshift.node.schedulable, am I correct that this is from openshift_facts and accurately reflects what the system was when facts were last loaded? (provided we don't reload facts in the middle of this, which I can't see anywhere but is a little bit of a risk)

If this is correct you could likely simplify the check, was_schedulable = openshift.node.schedulable.

Or I might be missing something here.